hsds/basenode.py

```diff
@@ -654,6 +784,12 @@ def baseInit(loop, node_type):
         log.info("running in kubernetes")
         app["is_k8s"] = True
 
+    # check to see if we are running in a DCOS cluster
+    is_dcos = os.environ.get('MARATHON_APP_ID')
```

Comment:
Won't this throw an exception for non-dcos?

Reply:
fixed